faithcomesbyhearing/dbp
# FCBH-1676 Create new Chapter Detail Endpoint

# Description
- Added bibles/{bible_id}/copyright copyrigth endpoint
- Added bibles/{bible_id}/chapter chapter endpoint
- Fixed api_status web endpoint

## Issue Link
Story: [FCBH-1676](https://fullstacklabs.atlassian.net/browse/FCBH-1676)  

## How Do I QA This
- Run `http://dbp.test/api/bibles/ENGESV/chapter?api_token={API_TOKEN}&v=4&key={API_KEY}&book_id={BOOK_ID}&chapter={CHAPTER}&copyrights={TRUE/FALSE}&drama={TRUE/FALSE/NULL}&zip={TRUE/FALSE}` and verify that the endpoint work as expected

## Checklist
